java/ql/test/query-tests/Telemetry/UnsupportedExternalAPIs/UnsupportedExternalAPIs.expected

```diff
@@ -0,0 +1,6 @@
+| java.io;?;PrintStream;println;(Object) | 3 |
+| java.lang;?;Object;Object;() | 2 |
```

Comment:
Curious: where's the second `new Object()` call?
Also, argumentless constructors seem like something we should filter: they can't have summaries and can't be sinks and I guess they aren't likely sources?

Reply:
Done